src/stdlib_experimental_io.f90

```diff
@@ -268,4 +269,39 @@ logical function whitechar(char) ! white character
 end if
 end function
 
+integer function open(filename, mode) result(u)
```

Comment:
As written now, this 'open' function will be only for text files. However, Fortran also supports other types of files. So maybe we could name this function `opentxt`? (to be similar to `savetxt`, and `loadtxt`)

Reply:
After having discusssed with @certik I am ok with `open`.